packages/lesswrong/components/events/modules/VirtualProgramCard.tsx

```diff
@@ -112,12 +114,12 @@ const VirtualProgramCard = ({program, classes}: {
   const now = moment()
   let deadline = now.date(28).day(0)
   // If that Sunday is in the past, use next month's 4th Sunday.
-  if (deadline.isBefore(moment())) {
-    deadline = now.add(1, 'months').date(28).day(0)
+  if (deadline.isBefore(now)) {
+    deadline = moment(now).add(1, 'months').date(28).day(0)
   }
   
-  // VP starts 15 days after the deadline, on a Monday
-  const startOfVp = moment(deadline).add(15, 'days')
+  // VP starts 22 days after the deadline, on a Monday
+  const startOfVp = moment(deadline).add(22, 'days')
```

Comment:
Also updated this to match the current dates listed on the VP website, but I feel like they randomly change this logic (or at least they won't give me a straight answer about what heuristics they use to decide these dates) so will prob be wrong again in the future 🤷‍♀️